### ISSUE TITLE
Carpenter takes no fluids

### ISSUE THREAD

**Issue by Schlauchiboy** [ForestryMC/ForestryMC#2497]
I want to craft the impregnated casing using the Carpenter. The wood is placed in the right order and there is 32 Wood in the Carpenter. But now the seed oil will not get into the Carpenter. Tried a few cables for fluids but no one is working. Also tried to connect the machine directy to the Carpenter. But this does not work, too. The fluid will stay in the cables. Am i doing something wrong?? Playing All the mods 3 remix with forestry version 5.8.2.394 for minecraft 1.12.2. Thanks for help...

- Now used the newest forestry version and it does not work, too.
- Carpenter only accepts water from the pipes. no other fluids

**Comment by Nedelosk** [comment]
Have you tried to use fluid container items to get fluids into the carpenter?

**Comment by Schlauchiboy** [comment]
Yes. I tried nearly everything to put the fluid into the machine. As I said, it is working perfectly with water and the water can be transported with every pipe I want. So I don't think there is a problem with the mods of the cable. Maybe there is a config in which you can choose if other fluids are accepted to get into the carpenter? But I don't know. Thanks for your answer.
